### PR TITLE
Reformat Getting started page on docs site

### PR DESCRIPTION
**Pull request by kelbrown20**
**Description:** Separating Installing and other Instructlab steps on the docs site. Good to have the different install methods, but after Installing, most of the other workflow items should be the same. 

Proposed map for "Getting started"

```
Getting started
- Install on Linux AMD
- Install on Linux Nvidia 
- Install on Mac
- Initialize InstructLab 
- Download models
- Intro Serve and chat
```

This PR also moved the other md files in getting_started to a different section. Creating a different PR for populating the "Adding data to a model" section

### CODE REVIEW

**Comment by instructlab-inviter** [comment]
Can you update the `mkdocs.yaml` with the nav changes too?

**Comment by jjasghar** [comment]
Can you update the mkdocs.yaml with the nav changes too?

File: docs/getting-started/download_models.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+```shell
+ilab model download
+```
+
+`ilab model download` downloads a compact pre-trained version of the [model](https://huggingface.co/instructlab/) (~4.4G) from HuggingFace:
+
+```shell
+(venv) $ ilab model download
+Downloading model from Hugging Face: instructlab/merlinite-7b-lab-GGUF@main to /Users/USERNAME/Library/Caches/instructlab/models...
```
Comment (jjasghar) [inline docs/getting-started/download_models.md:19]: We should update this to granite right?

File: docs/getting-started/download_models.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+```
+
+*Example output of `ilab model list` after `ilab model download`*
+
+```shell
+(venv) $ ilab model list
++------------------------------+---------------------+--------+
+| Model Name                   | Last Modified       | Size   |
++------------------------------+---------------------+--------+
+| merlinite-7b-lab-Q4_K_M.gguf | 2024-08-01 15:05:48 | 4.1 GB |
```
Comment (jjasghar) [inline docs/getting-started/download_models.md:52]: Add per the granite comment above ^^

File: docs/getting-started/initilize_ilab.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+4) When prompted, provide the path to your default model. Otherwise, the default of a quantized [Merlinite](https://huggingface.co/instructlab/merlinite-7b-lab-GGUF) model will be used - you can download this model with `ilab model download` (see below).
+
+   ```shell
+   (venv) $ ilab config init
+   Welcome to InstructLab CLI. This guide will help you set up your environment.
+   Please provide the following values to initiate the environment [press Enter for defaults]:
+   Path to taxonomy repo [taxonomy]: <ENTER>
+   `taxonomy` seems to not exists or is empty. Should I clone https://github.com/instructlab/taxonomy.git for you? [y/N]: y
+   Cloning https://github.com/instructlab/taxonomy.git...
+   Path to your model [/Users/USERNAME/Library/Caches/instructlab/models/merlinite-7b-lab-Q4_K_M.gguf]: <ENTER>
```
Comment (jjasghar) [inline docs/getting-started/initilize_ilab.md:51]: This is also assuming Mac, the `/Users/` is specific to the Mac, installation, maybe we should make that clear here.

File: docs/getting-started/initilize_ilab.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+   ```
+
+5) When prompted, please choose a train profile. Train profiles are GPU specific profiles that enable accelerated training behavior. **YOU ARE ON MacOS**, please choose `No Profile (CPU, Apple Metal, AMD ROCm)` by hitting Enter. There are various flags you can utilize with individual `ilab` commands that will allow you to utilize your GPU if applicable.
+
+   ```shell
+    Welcome to InstructLab CLI. This guide will help you to setup your environment.
+    Please provide the following values to initiate the environment [press Enter for defaults]:
+    Path to taxonomy repo [/home/user/.local/share/instructlab/taxonomy]:
```
Comment (jjasghar) [inline docs/getting-started/initilize_ilab.md:59]: This on the other hand is the Linux installation, `/home/user`

File: docs/getting-started/linux_nvidia.md [hunk 5]
```diff
@@ -1,10 +1,10 @@
 ---
-title: Getting Started with Linux NVidia
+title: Install on Linux NVidia
```
Comment (jjasghar) [inline docs/getting-started/linux_nvidia.md:2]: Not 100% sure the official way we're supposed to write "NVidia", we should confirm this so it's brand correct.

File: docs/getting-started/serve_and_chat.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+# 🍴 Serving the model
+
+- Serve the model by running the following command:
+
+```shell
+ilab model serve
+```
+
+erve a non-default model (e.g. Mixtral-8x7B-Instruct-v0.1):
```
Comment (jjasghar) [inline docs/getting-started/serve_and_chat.md:15]: We should convert this to Granite :) and `erve` isn't a word :)

File: docs/getting-started/serve_and_chat.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+```shell
+source venv/bin/activate
+ilab model chat --model models/mixtral-8x7b-instruct-v0.1.Q4_K_M.gguf
+```
+
+Please note that usage of `--model` necessitates that the existing server has that model. If not, you must exit the server. `--model` in `ilab model chat` has the ability to start a server on your behalf with the specified model if one is not already running on the port.
+
+Before you start adding new skills and knowledge to your model, you can check its baseline performance by asking it a question such as `what is the capital of Canada?`.
+
+> [!NOTE]
```
Comment (jjasghar) [inline docs/getting-started/serve_and_chat.md:72]: This needs to be converted to `!!! note` formatting.

**Review by jjasghar — CHANGES_REQUESTED** [review]
Made some suggestions and updates :)

File: docs/getting-started/linux_nvidia.md (hunk 5 above)
Comment (kelbrown20) [reply docs/getting-started/linux_nvidia.md:2]: So I believe the brand offical way to write it is NVIDIA.

File: docs/getting-started/download_models.md (hunk 1 above)
Comment (kelbrown20) [reply docs/getting-started/download_models.md:19]: Im pretty sure the default is still merlinite, I'll change it to granite once that is updated.

File: docs/getting-started/initilize_ilab.md (hunk 3 above)
Comment (kelbrown20) [reply docs/getting-started/initilize_ilab.md:51]: Good point! I'll clarify that here!

**Review by jjasghar — APPROVED** [review]

**Pull request merged**